README.md

```diff
@@ -41,6 +41,15 @@ profile.followees().then(followees => {
 // Can lookup 'activityId' from 'currentActivityId' of profile() response
 profile.giveRideOn(otherRiderId, activityId);
 
+// Retrieve the goals you have
+goals = profile.getGoals();
+goals.then(goal=> {
+    console.log(goal); //JSON array of goals
+})
```

Comment:
To keep the style similar for the above, I'd write this as:
````
profile.goals().then(goals => {
    console.log(goals); //JSON array of goals
})
````

Reply:
Done